test-integration/apps/beacon-consumer/app.js

```diff
@@ -9,8 +9,23 @@ module.exports = ({ beaconFilePath, logger }) => {
 
     app.use(morgan('dev'));
 
-    return app.post('/', pixelParser, analyticsHandler({
+    const handler = analyticsHandler({
         beaconFilePath,
         logger,
-    }));
+    });
+
+    // for "search" event type
+    app.post('/', pixelParser, handler);
+
+    // For all other event types
+    [
+        'autoSearch',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/groupby/gb-tracker-client/pullRequest?prid=7996839)
```suggestion
        "autoSearch",
```